src/calendar/day/index.tsx

```diff
@@ -12,13 +9,14 @@ import {SELECT_DATE_SLOT} from '../../testIDs';
 import {DateData} from '../../types';
 import BasicDay, {BasicDayProps} from './basic';
 import PeriodDay from './period';
+import {isEqual, omit, some} from 'lodash';
```

Comment:
No. The reason we import each separately is to control the import package size. This is how we do it all through the lib, please don't change it.